### ISSUE TITLE
Bug: Insufficient contrast in dark theme

### ISSUE THREAD

**Issue by amagnolo** [heyman/heynote#491]
**Describe the bug**
Colors in dark theme have insufficient contrast.

**To Reproduce**
1. Select dark theme
2. Write something that gets colored, like a heading in markdown
3. See that the color contrast between text and background is below WCAG recommendations

**Expected behavior**
A sufficient contrast to make the text more readable.

**Screenshots**
Screenshots can be found in this related issue: https://github.com/heyman/heynote/issues/235#issuecomment-4467099278

**Desktop:**
 - OS: Linux
 - Version 2.9.0

**Additional context**
I originally reported this problem in the related issue, but I understand that adding full CSS customization is more complicated and overkill for this accessibility problem, that can be fixed by adjusting the default colors.
